pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/prometheus/alert-rules/index.md

```diff
@@ -0,0 +1,167 @@
+
+```json
+{
+  "prometheus": {
+    "alert_rules_repository": {
+      "config_repository": {
+        "url": "https://github.com/company/alert-rules",
+        "path": "/production"
+      }
+    }
+  }
+}
+```
+
+<p class="message--important"><strong>IMPORTANT: </strong>Make sure that each of the alert rule files has suffix <code>.yml</code>
+and is a direct child of the folder (that's not nested under another folder).</p>
+
+The Git repository can be either public or private.
+If the Git repository is private, you will need to configure the credentials to access the Git repository (see below).
+
+<p class="message--note">
+<strong>NOTE:</strong> Mesosphere maintains a Git repository with Prometheus alerting rule configurations at <code>https://github.com/dcos/prometheus-alert-rules</code>.
+If you would like to use these Prometheus alerting rules, you should set <code>prometheus.alert_rules_repository.config_repository.url</code>
+to <code>https://github.com/dcos/prometheus-alert-rules</code> and <code>prometheus.alert_rules_repository.config_repository.path</code> to <code>/rules</code>.
+As it's a public repository, there is no need to set up the <code>credentials</code>.
+</p>
+
+```json
+{
+  "prometheus": {
+    "alert_rules_repository": {
+      "config_repository": {
+        "url": "https://github.com/dcos/prometheus-alert-rules",
+        "path": "/rules"
+      }
+    }
+  }
+}
+```
+
+## Create secrets for Git repository credentials
+
+If the Git repository containing the alert rules is private, you will need to configure the secrets first.
+Currently, the following Auth types are supported.
+
+<p class="message--important"><strong>IMPORTANT: </strong>If the Git repository contains submodules, all the submodules must use the same Auth type as what's used for the repository. For example, if HTTP Auth is configured for the Git repository, all of the submodules in the repository's <code>.gitmodules</code> file must use the <code>https</code> URL scheme.</p>
+
+### HTTP Auth
+
+```bash
+dcos security secrets create --value=<GITHUB_USERNAME> githubusername-secret
+dcos security secrets create --value=<GITHUB_PASSWORD> githubpassword-secret
+```
+
+For GitHub, the password can be the API token.
+
+Create a custom option file (`options.json`) like the following.
+You may omit the `credentials` section if the Git repository is public.
+
+```json
+{
+  "prometheus": {
+    "alert_rules_repository": {
+      "url": "https://github.com/company/alert-rules",
+      "path": "/production",
+      "credentials": {
+        "username_secret": "githubusername-secret",
+        "password_secret": "githubpassword-secret"
+      }
+    }
+  }
+}
+```
+
+### SSH Auth
+
+```bash
+dcos security secrets create -f <PATH_TO_PRIVATE_KEY> gitsshkey-secret
+```
+
+For GitHub, you must add the [Deployment Key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) (that's the public key) to the repository.
```

Comment:
The change from "that is" -> "that's" here, while probably technically correct, is awkward and we can see Vale also complains.

Please change back to "(that is, the public)", or just "(the public key)